/**:
  ros__parameters:
    names: [ "J1_base", "J2_shoulder", "J3_elbow", "J4_wrist" ]
    families: [ "HEBIArm" ]
    gains_package: "hebi_description"
    gains_file: "config/gains/A-2084-01_gains.xml"
    hrdf_package: "hebi_description"
    hrdf_file: "config/hrdf/A-2084-01.hrdf"
    home_position: [ -1.0, 0.0, -2.0, 0.0 ]
